charts/timed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.9.1
+version: 0.10.6
```

you only need to bump the version once:
```suggestion
version: 0.10.0
```